### PR TITLE
created data models

### PR DESCRIPTION
**Pull request by JacobRintjema**

### CODE REVIEW

File: backend/models/user-model.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export class User {
+  userID : number;
+  firstName : string;
+  lastName : string;
+  height : number;
+  weight : number;
+  exercise : number;
+  gender : string;
+  dietPreferences : string;
```
Comment (johnswyoon) [inline backend/models/user-model.ts:9]: I think we may want an array of strings `dietPreferences: Array<String>`
For example, if someone needs vegan and halal dietary options.

File: backend/models/recipe-model.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export class Recipe {
+  recipeID : number;
+  recipeTitle : string;
+  recipeIngredients: Array<string>;
```
Comment (johnswyoon) [inline backend/models/recipe-model.ts:4]: According to the Edamam API i think the ingredients are an array of objects
Screenshot 2023-07-05 at 4 49 58 PM

File: backend/models/recipe-model.ts
Comment (johnswyoon) [inline backend/models/recipe-model.ts:1]: Can you add a field for `images` so we can add a thumbnail in the frontend <3

File: backend/models/fridge-model.ts
Comment (johnswyoon) [inline backend/models/fridge-model.ts:1]: Should we add a "createdOn" date so we can sort by that? Maybe for fridge and items?

**Review by johnswyoon — CHANGES_REQUESTED** [review]
Just added a few comments

File: backend/models/user-model.ts (hunk 1 above)
Comment (JacobRintjema) [reply backend/models/user-model.ts:9]: good point

File: backend/models/recipe-model.ts (hunk 2 above)
Comment (JacobRintjema) [reply backend/models/recipe-model.ts:4]: ty

File: backend/models/recipe-model.ts
Comment (JacobRintjema) [reply backend/models/recipe-model.ts:1]: what data type? how would that work?

File: backend/models/fridge-model.ts
Comment (JacobRintjema) [reply backend/models/fridge-model.ts:1]: you mean in addition to the expirationDate in Item-model?

**Pull request merged**